src/video.test.ts

```diff
@@ -30,7 +35,7 @@ describe('parseVideo', () => {
 				},
 			],
 			contentAtomTypeData: {
-				atomId: 'atomId2',
+				atomId: atomId,
```

Comment:
If the object key has the same name as the value variable you can shorthand it like so:
```
contentAtomTypeData: {
	atomId,
	atomType: 'someAtomType',
}
```

Reply:
Thanks Fares for pointing it out